packages/ckeditor5-coremedia-link/src/linktarget/config/LinkTargetDefaultRuleDefinition.ts

```diff
@@ -0,0 +1,68 @@
+export type TargetDefaultRuleDefinition = TargetDefaultRuleDefinitionWithFilter | TargetDefaultRuleDefinitionWithType;
+
+/**
+ *
+ */
+export interface TargetDefaultRuleDefinitionWithFilter {
+  /**
+   * A filter rule, used to check whether the given default target should be applied to the link.
+   *
+   * Please note: Since content links and external links are the most common types to
+   * have a particular link target, you can also use the "type" property to configure them directly.
+   *
+   * This filter would apply to all content links:
+   * @example
+   * ```
+   * filter: (url: string) => (url ? url.startsWith("content:") : false);
+   * ```
+   *
+   * @param url - the url of the link
+   * @returns whether to apply the target to the link
+   */
+  filter: (url: string) => boolean;
+
+  /**
+   * The link target, that should be applied to all inserted links, that match
+   * the filter rule.
+   */
+  target: string;
+}
+
+export interface TargetDefaultRuleDefinitionWithType {
```

Comment:
According to the interface documentation for the "withFilter" I would expect something similar for the "withType" interface.